### DISCUSSION TITLE
Why Push-based execution improves cache efficiency ?

### DISCUSSION THREAD

**Discussion by IronsDu** [duckdb/duckdb#2992] [Q&A]
Hello, I see http://justinjaffray.com/query-engines-push-vs.-pull/ have this statement:
` Push-based execution improves cache efficiency`.

I don't know the exact reason. Can you explain it? 
Thank you.

---
in addition: this execution not have query compilation.

**Comment by Alex-Monahan** [comment]
Thanks for your interest in DuckDB! I don't have the exact answer to your question, but here is a [video that describes DuckDB's push based execution](https://www.youtube.com/watch?v=MA0OsvYFGrc)!

**Comment by IronsDu** [comment]
@Alex-Monahan Can you share this PPT to me ? irons.du@gmail.com
